src/Resources/app/administration/src/module/findologic-module/snippet/en-GB.json

```diff
@@ -12,7 +12,7 @@
     "settingForm": {
       "salesChannelTitle": "Sales Channel & Language",
       "directIntegration": {
-        "title": "Direct Integration"
+        "title": "Direct Integration / Instant Frontend"
```

Comment:
There was no need to translate it to German, but we still need that exact value for German.